### DISCUSSION TITLE
Pending Updates can't be found

### DISCUSSION THREAD

**Discussion by grungle25** [rickysarraf/apt-offline#189] [Q&A] [answered]
Hi

After installing my apt-offline produced ZIP file I had 256 updates to apply immediately.  I then used apt update --fix-missing and it updated a bunch of stuff but also produced a bunch of errors.   I now have 59 updates that can be applied immediately but if I try to upgrade them using apt update or apt update --fix-missing.

I produced another SIG file and repeated the process but the 59 updates remain.  I'm assuming I messed something up early on so was hoping there was a way to 'reset' the update process so that the OS (Ubuntu 20.04) doesn't think there are any updates available.  Any ideas?

Thanks

**Comment by rickysarraf** [comment, marked as answer]
This is wrong step. You're not supposed to run `apt update`. Because that command option assumes that the machine has internet connectivity.

Please go through the howto [here](https://github.com/rickysarraf/apt-offline/wiki) and also the manpage. That should give you the right instructions.

There's also a video linked in the document where I demonstrate the use.

**Reply by grungle25** [reply]
Thanks for the reply.  I wish I had read your instructions first rather than some random website.  :)  Thanks for the great product!
